src/lib/validate-raw-data.js

```diff
@@ -0,0 +1,60 @@
+import { database_pool } from './database.js';
+import validator from '@openactive/data-model-validator';
+
+/*
+ * TODO This will run only one check at a time - add something so it runs several at once!
+ * But I want to discuss with others best way to do that first
+ */
+async function validate_raw_data_all() {
+
+    const client = await database_pool.connect();
+    try {
+        while(true) {
```

Comment:
Just wondering if you're expecting this to be OK to fail?

Reply:
Not confident, but this applies to many other places and we need a general review of handling error conditions https://github.com/openactive/conformance-services/issues/26